crates/astria-bridge-withdrawer/src/bridge_withdrawer/ethereum/watcher.rs

```diff
@@ -641,23 +648,23 @@ mod tests {
 
         let value = 1_000_000_000.into();
         let recipient = crate::astria_address([1u8; 20]);
-
         let bridge_address = crate::astria_address([1u8; 20]);
-        let denom = default_native_asset();
+        let denom = "transfer/channel-0/utia".parse::<Denom>().unwrap();
```

Comment:
```suggestion
        let denom = "nria".parse::<Denom>().unwrap();
```

arbitrary anyways since it's a test, but this matches what was there before